test/unit/tokenprocessing.spec.js

```diff
@@ -1,88 +1,81 @@
 /* global $,postscribe,test,ok,start,setOptions */
 /* eslint-disable no-var */
+import postscribe from '../../dist/postscribe';
 
-$(document).ready(function() {
-
-  module('token processing');
-  setOptions({});
-
-  function testBeforeWriteTokenCalled(name, html, type, expectedAmount) {
-    test(name, function() {
-      var amount = 0;
-      var div = document.createElement('div');
+describe('token processing', function() {
+  const testBeforeWriteTokenCalled = (name, html, type, expectedAmount) => {
```

Comment:
I'd tend to refactor this a bit differently and return a `done => {}` function, so the call is more like:
```
it('beforeWriteToken is called for div', callsBeforeWriteToken(1, '<div>Test</div>', 'div'));
```

Also, looks like you could use `sinon.spy()` or `sinon.mock()` for this.